.github/workflows/ci.yml

```diff
@@ -46,11 +32,13 @@ jobs:
         run: brew update && brew reinstall libev openssl@3 # Openssl is a workaround for https://github.com/ocaml/opam-repository/issues/19676
         if: runner.os == 'macOS'
 
-      - name: Install opam dependencies
-        run: opam install --deps-only --with-test .
+      - name: Setup Dune
+        uses: ocaml-dune/setup-dune@v1
+        with:
+          version: 3.20.1
```

Comment:
Is it necessary to pin a specific version? Why not `"dev"`?

> The `version` can have the following special values:
>
>    `dev` for the Developer Preview, which is the default at the moment since dune package management is still an experimental feature,
>    `latest` for the latest stable release.

I'm surprised that this actually works since this pins a specific version, rather than the dev preview, or has dune pkg been released in this stable version?

Reply:
But also, yes, `dune pkg` is released in stable

Reply:
going with `latest` sounds good to me